app/templates/components/public/session-item.hbs

```diff
@@ -12,17 +12,17 @@
     <br>
     <div class="ui grid">
       <div class="row">
-        <div class="column twelve wide">
+        <div class="column four wide">
```

Comment:
reduction from twelve to four seems a bit extreme and will cause UI issues when there are a large number of speakers as pointed by @shubham3211 in the screenshit., instead give  `right floated ui segment` a try.
https://semantic-ui.com/elements/segment.html#floated